simulation_parameters/microbe_stage/membranes.json

```diff
@@ -27,7 +29,8 @@
         "cellWall": true,
         "normalTexture": "res://assets/textures/CellWallGradient.png",
         "damagedTexture": "res://assets/textures/CellWallGradientDamaged.png",
-        "movementFactor": 0.8,
+        "movementFactor": 0.4,
```

Comment:
The movement factor has balancing implications and I don't think it should be included in this PR
This is why I wanted the movement wigglyness to be a separate thing